Worldepth/app/src/main/cpp/SLAM-include/System.h

```diff
@@ -58,6 +58,7 @@ namespace SLAM
         // Initialize the SLAM system. It launches the Local Mapping, Loop Closing and Viewer threads.
         System(const string &strVocFile, const string &strSettingsFile);
 
+        void DeleteORBVocab();
```

Comment:
Do we need this? There is no implementation in the System.cpp, so I'm assuming this is an accident

Reply:
yeah ok will delete